File: java/2.methods/parameters.java
public class parameters {
  static void namer(String name) {
    System.out.println("Hello, " + name + "!");
  }
  static int add(int x, int y) {
    return x + y;
  }
  public static void main(String[] args) {
    namer("Mevia");
    System.out.println(add(5, 10));
  }
}

File: cs/2.methods/parameters.cs
using System;

namespace parameter
{
  class pm
  {
    static void greet(string fname)
    {
      Console.WriteLine("Hello, " + fname + ".");
    }
    static void Main(string[] args)
    {
      greet("Mevia");
      greet("Alex");
    }
  }
}
